lib/broadside/configuration/invalid_configuration.rb

```diff
@@ -5,9 +5,5 @@ module InvalidConfiguration
     def method_missing(m, *args, &block)
       warn "Unknown configuration '#{m}' provided, ignoring."
     end
-
-    def respond_to_missing?(method, include_private = false)
```

Comment:
i thought about this more and i think we don't need to do this, no matter what hound says. 
Realistically `Configuration` isn't responding to these calls, and there's no reason to inform any other part of the ruby interpreter that it can respond to them.